src/Utilities/PaymentMethodCollectUtils.res

```diff
@@ -171,56 +171,68 @@ let getPaymentMethodDataFieldKey = (key: paymentMethodDataField): string =>
   | VenmoMobNumber => "venmo.phoneNumber"
   }
 
-let getPaymentMethodDataFieldLabel = (key: paymentMethodDataField): string =>
+let getPaymentMethodDataFieldLabel = (
+  key: paymentMethodDataField,
+  localeString: LocaleStringTypes.localeStrings,
+): string =>
   switch key {
-  | CardNumber => "Card Number"
-  | CardExpDate => "Expiry Date"
-  | CardHolderName => "Cardholder Name"
-  | ACHRoutingNumber => "Routing Number"
-  | ACHAccountNumber | BacsAccountNumber => "Account Number"
-  | BacsSortCode => "Sort Code"
-  | SepaIban => "International Bank Account Number (IBAN)"
-  | SepaBic => "Bank Identifier Code (Optional)"
-  | PixId => "Pix ID"
-  | PixBankAccountNumber => "Bank Account Number"
-  | PaypalMail => "Email"
-  | PaypalMobNumber | VenmoMobNumber => "Phone Number"
-  | SepaCountryCode => "Country Code (Optional)"
+  | CardNumber => localeString.cardNumberLabel
+  | CardExpDate => localeString.validThruText
+  | CardHolderName => localeString.cardHolderName
+  | ACHRoutingNumber => localeString.formFieldACHRoutingNumberLabel
+  | ACHAccountNumber | BacsAccountNumber => localeString.accountNumberText
+  | BacsSortCode => localeString.sortCodeText
+  | SepaIban => localeString.formFieldSepaIbanLabel
+  | SepaBic => localeString.formFieldSepaBicLabel
+  | PixId => localeString.formFieldPixIdLabel
+  | PixBankAccountNumber => localeString.formFieldBankAccountNumberLabel
+  | PaypalMail => localeString.emailLabel
+  | PaypalMobNumber | VenmoMobNumber => localeString.formFieldPhoneNumberLabel
+  | SepaCountryCode => localeString.formFieldCountryCodeLabel
   | ACHBankName
   | BacsBankName
   | PixBankName
-  | SepaBankName => "Bank Name (Optional)"
+  | SepaBankName =>
+    localeString.formFieldBankNameLabel
   | ACHBankCity
   | BacsBankCity
-  | SepaBankCity => "Bank City (Optional)"
+  | SepaBankCity =>
+    localeString.formFieldBankCityLabel
   | CardBrand => "Misc."
```

Comment:
Is there no need to add locale support for Miscellaneous?

Reply:
This is basically a meta field, this will not be rendered on web or sent in the API